ui/store/actionConstants.ts

```diff
@@ -145,3 +145,28 @@ export const SHOW_KEYRING_SNAP_REMOVAL_RESULT =
 export const HIDE_KEYRING_SNAP_REMOVAL_RESULT =
   'HIDE_KEYRING_SNAP_REMOVAL_RESULT';
 ///: END:ONLY_INCLUDE_IF
+
+// Metamask Notifications
+export const PERFORM_SIGN_IN = 'PERFORM_SIGN_IN';
```

Comment:
I'll need you to expand on these actions... why are they needed?

Reply:
@matteoscurati double check, see if can remove or not.

Reply:
Removed!